Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/ProjectStatsController.java

```diff
@@ -34,92 +54,162 @@
+		{
+			if(network.getStatus().equals(status) || status.equals("SUBMITTED"))
+			{
+				if(contributionCount.containsKey(sdf.format(network.getCreatedDate())))
+				{
+					contributionCount.put(sdf.format(network.getCreatedDate()), contributionCount.get(sdf.format(network.getCreatedDate()))+1);
+				}
+				else
+				{
+					contributionCount.put(sdf.format(network.getCreatedDate()), 1);
+				}
+			}
+		}
+		for(Entry<String, Integer> entry : contributionCount.entrySet())
+		{
+			JSONObject jsonObject = new JSONObject();
+			jsonObject.put("date", entry.getKey());
+			jsonObject.put("count", entry.getValue());
+			contributionsJson.put(jsonObject);
+		}
+		return contributionsJson;    	
+	}
+
+	/**
+	 * This method gives the visualization of how often concepts appear in the
+	 * networks
+	 * 
+	 * @author Bharath Srikantan & Ajay Modi
+	 * @param projectUnixName
+	 *            The project unix name
+	 * @param model
+	 *            Model
+	 * @return view
+	 * @throws JAXBException
+	 * @throws QuadrigaStorageException
+	 * @throws JSONException 
+	 */
+
+	@RequestMapping(value = "sites/{projectUnixName}/statistics", method = RequestMethod.GET)
+	public String showProjectStatistics(
+			@PathVariable("projectUnixName") String projectUnixName, Model model, Principal principal)
+					throws JAXBException, QuadrigaStorageException, JSONException {
+
+		IProject project = projectManager.getProjectDetailsByUnixName(projectUnixName);
+
+		if (project == null) {
+			return "auth/accessissue";
+		}
+
+		String projectId = project.getProjectId();
+		List<INetwork> networks = networkmanager.getNetworksInProject(projectId);
+		List<IConceptStats> conceptsWithCount = null;
+		JSONArray submittedNetworkCount = null, approvedNetworkCount = null, rejectedNetworkCount  = null, workspaceCount = null;
+		String submittedNetworks = null, approvedNetworks = null, rejectedNetworks = null;
+		
+		if (!networks.isEmpty()) {
+			conceptsWithCount = projectStats.getConceptCount(networks);
+
+			try {
+				JSONArray jArray = null;
+				int cnt = getCount(conceptsWithCount);
+				jArray = getTopConceptsJson(conceptsWithCount.subList(0, cnt),
+						cnt);
+				String jsonData = jArray.toString(); 
+				model.addAttribute("labelCount", jsonData);
+				model.addAttribute("networkid", "\"\"");
+
+			} catch (JSONException e) {
+
+				StringBuffer errorMsg = new StringBuffer();
+				model.addAttribute("show_error_alert", true);
+				errorMsg.append(e.getMessage());
+				errorMsg.append("\n");
+				model.addAttribute("error_alert_msg", errorMsg.toString());
+			}
+
+			submittedNetworkCount= getContributionCountByStatus(networks,SUBMITTED);
+			approvedNetworkCount= getContributionCountByStatus(networks,APPROVED);
+			rejectedNetworkCount= getContributionCountByStatus(networks,REJECTED);
+			
+			submittedNetworks = submittedNetworkCount.toString();
+			approvedNetworks = approvedNetworkCount.toString();
+			rejectedNetworks = rejectedNetworkCount.toString();

```

Comment:
Instead of having another set of variables, just call 'toString' in when you add the counts to the model.